lib/screens/friends/friends_profile.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:petmo/models/user/user_details.dart';
+import 'package:petmo/screens/friends/friends_list.dart';
+import 'package:petmo/screens/pet/pet_screen.dart';
+
+import '../style.dart';
+
+class Friendsprofile extends StatelessWidget {
```

Comment:
Typical naming convention for screens would be `friends_profile_screen.dart` and then the class called `FriendsProfileScreen`, just so that it is obvious what the file / class is.